brickflow/codegen/databricks_bundle.py

```diff
@@ -617,6 +617,22 @@ def proj_to_bundle(self) -> DatabricksAssetBundles:
             / self.env
         )
 
+        bundle_suffix = config(
+            BrickflowEnvVars.BRICKFLOW_WORKFLOW_SUFFIX.value,
+            default=None,
+        )
+
+        bundle_prefix = config(
+            BrickflowEnvVars.BRICKFLOW_WORKFLOW_PREFIX.value,
+            default=None,
+        )
+
+        if bundle_prefix is not None:
+            bundle_root_path = bundle_root_path / bundle_prefix
+
+        if bundle_suffix is not None:
+            bundle_root_path = bundle_root_path / bundle_suffix
```

Comment:
if user gives both prefix and suffix then the path would be as below

`Users/${workspace.current_user.userName}/.brickflow_bundles/test-project/local/_prefix/_suffix`

is this expected functionality?

Reply:
nvm, I see that in your test case example!